resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -222,9 +222,15 @@ cloudwatch.logs.open_in_editor=Open in Editor
 cloudwatch.logs.open_in_editor_failed=Open in editor failed
 cloudwatch.logs.opening_in_editor=Opening Stream {0} in editor
 cloudwatch.logs.query=Open Query Editor
-cloudwatch.logs.query.form.ok_Button=Apply
+cloudwatch.logs.query.form.ok_Button=Execute
 cloudwatch.logs.query_editor_title=Query Log Groups
+cloudwatch.logs.query_name=Query name is missing
+cloudwatch.logs.query_not_saved=Query could not be saved. Try saving with a different name.
```

Comment:
Can this be reworded to "Query with this name exists", also should we allow this? Shouldn't I be able to try out a query then update it? Maybe with Confirmation (fine to add a TODO)?